root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1787,3 +1776,118 @@ test('Cleanup', function (t) {
 
     t.end();
 });
+
+test('URL cleanup component: auto-select, clean-up, and validation', {}, function (t) {
+    const test_data = [
+        // AllMusic
+        {
+            i_url: 'http://www.allmusic.com/album/here-comes-the-sun-mw0002303439/releases',
+            i_ent: 'release_group',
+            match: 'allmusic',
+            clean: 'http://www.allmusic.com/album/mw0002303439',
```

Comment:
So I propose the following (lengthy but self-explanatory) (might be less readable though) renaming:
- `i_url` -> `input_url`
- `i_ent` -> `input_entity_type`
- `match` -> `expected_relationship_type`
- `clean` -> `expected_clean_url`
- `i_rel` -> `input_relationship_type`
- `valid` -> `only_valid_entity_types`
- `wrong` -> `only_wrong_entity_types`